.infrastructure/pv.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pv-data
+  namespace: todoapp
+spec:
+  storageClassName: standard
+  persistentVolumeReclaimPolicy: Delete
+  accessModes:
+    - ReadWriteMany
+  capacity:
+    storage: 1Gi
+  hostPath:
+    path: app/data/
```

Comment:
Using `hostPath` for `PersistentVolumes` is not recommended for production environments due to potential data loss risks. Consider using a more robust storage solution, such as cloud provider's persistent storage or networked storage, to ensure data reliability and safety.